src/main.js

```diff
@@ -525,6 +527,49 @@ class MiniGraphCard extends LitElement {
       </svg>`;
   }
 
+  renderGridLines() {
+    const {
+      height, hours_to_show, show,
+    } = this.config;
+    const grid_line_type = show.grid_line_type ? show.grid_line_type : false;
+    const grid_line_frequency = show.grid_line_frequency ? show.grid_line_frequency : 2;
```

Comment:
grid_lines_ratio

Reply:
Shall we provide an additional safety that this value is numerical & integer?

Reply:
That's not a bad idea. I've added a isInteger check.